docs/concepts/tokens/fungible-tokens/index.md

```diff
@@ -1,93 +1,58 @@
-
-Generally, this means that the account that created the trust line is responsible for the reserve and the issuer is not. <!-- STYLE_OVERRIDE: is responsible for -->
-
-Trust lines are automatically deleted if both sides' settings are in the default state and the balance is 0. This means that, to delete a trust line, you need to:
-
-1. Send a [TrustSet transaction][] to set your settings to the defaults.
-2. Offload any positive balance you have on the trust line. You could do this by sending a [payment](../../payment-types/cross-currency-payments.md), or by selling the currency in the [decentralized exchange](../decentralized-exchange/index.md).
-
-If your balance is negative (you are the issuer) or the other side's settings are not in the default state, you cannot cause the trust line to be totally deleted, but you can make it so that it does not count towards your owner reserve by following the same steps.
-
-Since the **Authorized** setting cannot be turned off after it has been turned on, it does not count toward the trust line's default state.
-
-### Free Trust Lines
-[[Source]](https://github.com/XRPLF/rippled/blob/2df7dcfdebcb0cdbd030c1f4b09ac748af95659c/src/xrpld/app/tx/detail/SetTrust.cpp#L387-L407 "Source")
-
-Since trust lines are a powerful feature of the XRP Ledger, there is a special feature to make an account's first two trust lines "free".
-
-When an account creates a new trust line, if the account owns at most 2 items in the ledger including the new line, the account's owner reserve is treated as zero instead of the normal amount. This allows the transaction to succeed even if the account does not hold enough XRP to meet the increased reserve requirement for owning objects in the ledger.
-
-When an account owns 3 or more objects in the ledger, the full owner reserve applies.
-
-
-## See Also
-
-- **Concepts:**
-    - [Decentralized Exchange](../decentralized-exchange/index.md)
-    - [Rippling](rippling.md)
-- **References:**
-    - [account_lines method][] - Look up trust lines attached to a given account
-    - [gateway_balances method][] - Look up an issuer's total balance issued
-    - [RippleState object](../../../references/protocol/ledger-data/ledger-entry-types/ripplestate.md) - The data format for trust lines in the ledger's state data.
-    - [TrustSet transaction][] - The transaction to create or modify trust lines.
-
-{% raw-partial file="/docs/_snippets/common-links.md" /%}
+There are other strengths and weaknesses of each format, such as trust line tokens' ability to represent very large and very small quantities of the same token (greater than 63 orders of magnitude apart), or compatibility with legacy software.
```

Comment:
I would include this in the bullet list above. Also I'd take out the part about other strengths and weaknesses unless we plan on defining what those are here. As is, saying it is kind of an empty statement since we aren't pointing them to other docs are providing them the info here to help them choose.